rmgpy/rmg/main.py

```diff
@@ -684,20 +688,23 @@ def execute(self, **kwargs):
                             # Run a raw simulation to get updated reaction system threshold values
                             for index, reactionSystem in enumerate(self.reactionSystems):
                                 # Run with the same conditions as with pruning off
-                                reactionSystem.simulate(
-                                    coreSpecies = self.reactionModel.core.species,
-                                    coreReactions = self.reactionModel.core.reactions,
-                                    edgeSpecies = [],
-                                    edgeReactions = [],
-                                    surfaceSpecies = self.reactionModel.surface.species,
-                                    surfaceReactions = self.reactionModel.surface.reactions,
-                                    pdepNetworks = self.reactionModel.networkList,
-                                    modelSettings = tempModelSettings,
-                                    simulatorSettings = simulatorSettings,
-                                )
-                                self.updateReactionThresholdAndReactFlags(
-                                    rxnSysUnimolecularThreshold = reactionSystem.unimolecularThreshold,
-                                    rxnSysBimolecularThreshold = reactionSystem.bimolecularThreshold)
+                                if notResurrectedVec[index]:
```

Comment:
This causes reaction thresholds/flags to never be updated for reactors that were resurrected.
Shouldn't we reset the `notResurrectedVec[index]` flag at a certain point?
(What if there's only one reactor in the input file, and it got resurrected?)

Reply:
I'm not sure if I understand, notResurrectedVec is reset to all True every iteration of the while loop, so notResurrectedVec[index] should always equal the value of not resurrected during the iteration associated with reactionSystems[index].

Reply:
Yes, I see now. OK